superwall/src/main/java/com/superwall/sdk/models/paywall/PaywallPresentationInfo.kt

```diff
@@ -0,0 +1,16 @@
+package com.superwall.sdk.models.paywall
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class PaywallPresentationInfo(
+    // The presentation style of the paywall
+    val style: PaywallPresentationStyle,
+
+    // The condition for when a paywall should present.
+    val condition: PresentationCondition,
+
+    // The delay in milliseconds before switching from the loading view to
+    // the paywall view.
+    val loadingDelay: Long? = null
```

Comment:
This can be non-optional. Also let's change it to just `delay` to match iOS

Reply:
Updated, also changed how presentation is initialised in the `Paywall` itself by moving the init block to constructor.